process-engine/bpmn-studio
# ✨ Use Events for LET

**Changes:**

1. Use Events for LET
2. Refactor LET
3. Use Polling as Fallback

PR: #1412

> In order to use events a ProcessEngine > 8.0 will be needed, otherwise polling will be used.

## How can others test the changes?

- Open up the BPMN-Studios
- Run some processes
- **Notice that the LET is a bit faster and still colorizing everything correctly**

## PR-Checklist

Please check the boxes in this list after submitting your PR:
